### ISSUE TITLE
Method Post return value

### ISSUE THREAD

**Issue by NexusMTO** [s4cha/ws#9]
One question. I have problem with post

ws.post("/api/add_action",  params: ["user":"sasdas"]).then { json in
                            print(json["id"])
                        }

Error compilation

**Comment by s4cha** [comment]
@zephiro, this should be solved in version 1.2 (just got released)

``` swift
ws.post("/api/add_action", params: ["user":"sasdas"]).then { (json:JSON) in
print(json["id"])
}
```

Let me know how it goes :)

**Comment by Unstop-M** [comment]
Thanks, I upgrade.
